projects/kubernetes/cloud-provider-aws/README.md

```diff
@@ -0,0 +1,12 @@
+## **cloud-provider-aws**
+![Version](https://img.shields.io/badge/version-v1.22.3-blue)
+
+The [AWS cloud provider](https://github.com/kubernetes/cloud-provider-aws) provides the interface between a Kubernetes cluster and AWS service APIs. This project allows a Kubernetes cluster to provision, monitor and remove AWS resources necessary for operation of the cluster.
```

Comment:
Since we are only building/intending to use the credential provider portion of this project weshould include at information as part of the README.md.

### AWS Credential Provider
The AWS credential provider is a binary that is executed by kubelet to provide credentials for images in ECR.  Refer to the [credential provider extraction KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2133-out-of-tree-credential-provider) for more details.

Reply:
Done